web-marketplace/src/components/molecules/ProjectPageMetadata/ProjectPageMetadata.tsx

```diff
@@ -139,6 +147,16 @@ const ProjectPageMetadata: React.FC<React.PropsWithChildren<MetadataProps>> = ({
               metadata?.['@context'],
             )}
           />
+          <MetaDetail label={_(msg`region`)} value={bioregion} />
```

Comment:
TODO confirm whether region is `regen:bioregion` or not before merging @S4mmyb @erikalogie

Reply:
it's different, using `regen:region` now